2021-04-21 16:23:19,099 INFO    MainThread:7702 [wandb_setup.py:_flush():69] setting env: {}
2021-04-21 16:23:19,099 INFO    MainThread:7702 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-21 16:23:19,099 INFO    MainThread:7702 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210421_162319-1biz5nku/logs/debug.log
2021-04-21 16:23:19,099 INFO    MainThread:7702 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210421_162319-1biz5nku/logs/debug-internal.log
2021-04-21 16:23:19,099 INFO    MainThread:7702 [wandb_init.py:init():369] calling init triggers
2021-04-21 16:23:19,099 INFO    MainThread:7702 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-21 16:23:19,100 INFO    MainThread:7702 [wandb_init.py:init():418] starting backend
2021-04-21 16:23:19,100 INFO    MainThread:7702 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-21 16:23:19,114 INFO    MainThread:7702 [backend.py:ensure_launched():123] starting backend process...
2021-04-21 16:23:19,119 INFO    MainThread:7702 [backend.py:ensure_launched():128] started backend process with pid: 7730
2021-04-21 16:23:19,120 INFO    MainThread:7702 [wandb_init.py:init():423] backend started and connected
2021-04-21 16:23:19,121 INFO    MainThread:7702 [wandb_init.py:init():465] updated telemetry
2021-04-21 16:23:19,122 INFO    MainThread:7702 [wandb_init.py:init():484] communicating current version
2021-04-21 16:23:19,980 INFO    MainThread:7702 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-21 16:23:19,980 INFO    MainThread:7702 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-21 16:23:20,196 INFO    MainThread:7702 [wandb_init.py:init():522] starting run threads in backend
2021-04-21 16:23:20,225 INFO    MainThread:7702 [wandb_run.py:_console_start():1524] atexit reg
2021-04-21 16:23:20,225 INFO    MainThread:7702 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-21 16:23:20,226 INFO    MainThread:7702 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-21 16:23:20,228 INFO    MainThread:7702 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-21 16:23:20,228 INFO    MainThread:7702 [wandb_init.py:init():546] run started, returning control to user process
2021-04-21 16:23:34,646 INFO    MainThread:7702 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-21 16:23:34,656 INFO    MainThread:7702 [wandb_run.py:_restore():1466] restore
2021-04-21 16:23:36,870 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 569
}

2021-04-21 16:23:37,214 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 4672
}

2021-04-21 16:23:37,576 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4672
  total_bytes: 4674
}

2021-04-21 16:23:37,678 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4672
  total_bytes: 4674
}

2021-04-21 16:23:37,780 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:37,883 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:37,985 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:38,087 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:38,190 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:38,292 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:38,395 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:38,497 INFO    MainThread:7702 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4674
  total_bytes: 4674
}

2021-04-21 16:23:39,594 INFO    MainThread:7702 [wandb_run.py:_show_files():1838] logging synced files
